cmd/ocm/list/region/cmd.go

```diff
@@ -42,25 +43,61 @@ var Cmd = &cobra.Command{
 
 func init() {
 	fs := Cmd.Flags()
-	arguments.AddProviderFlag(fs, &args.provider)
+	fs.StringVar(
+		&args.provider,
+		"provider",
+		"",
+		"Lists the region for the specific cloud provider",
+	)
+
 	//nolint:gosec
 	Cmd.MarkFlagRequired("provider")
-	arguments.AddCCSFlagsWithoutAccountID(fs, &args.ccs)
+	fs.BoolVar(
+		&args.ccs,
+		"ccs",
+		false,
+		"Lists  the region specific to your cloud account",
+	)
+	fs.StringVar(
+		&args.awsAccessKeyId,
+		"aws-access-key-id",
+		"",
+		"AWS access key",
+	)
+
+	fs.StringVar(
+		&args.awsSecretAccessKey,
+		"aws-secret-access-key",
+		"",
+		"AWS Secret Access",
+	)
+
 }
 
 func run(cmd *cobra.Command, argv []string) error {
-	err := arguments.CheckIgnoredCCSFlags(args.ccs)
-	if err != nil {
-		return err
+	ccs := cluster.CCS{}
+	if args.ccs {
+		if args.awsAccessKeyId == "" {
+			return fmt.Errorf("--aws-access-key-id flag is mandatory for --ccs=true")
```

Comment:
What if user attempts this with --provider=gcp?
We don't have a similar `available_regions` API for GCP, but that but that asymmetry is not obvious to users.  I propose adding an error for `gcp` and `--ccs` combination, and since you're already tweaking the --help texts to better match this command, change the description of the --ccs flag to explain it's for AWS only.